src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,67 @@
+package duke;
+
+import duke.task.Task;
+
+import java.util.ArrayList;
+
+/**
+ * Represents a list of tasks.
+ */
+public class TaskList {
+
+    /** Array list of tasks. */
+    private final ArrayList<Task> tasks;
+
+    /**
+     * Creates a new and empty task list.
+     */
+    public TaskList() {
+        this.tasks = new ArrayList<>();
+    }
+
+    /**
+     * Gets the number of tasks in the list.
+     *
+     * @return Number of tasks in the list.
+     */
+    public int getNumOfTasks() {
+        return this.tasks.size();
+    }
+
+    /**
+     * Gets a task from the list.
+     *
+     * @param i Index of task stored in the list.
+     * @return Task from index i of the list.
+     */
+    public Task getTask(int i) {
```

Comment:
Perhaps consider changing the variable i into index to be more descriptive.

Reply:
Changed to index.